node/cli/src/command.rs

```diff
@@ -507,11 +512,14 @@ pub fn run() -> Result<()> {
 							_ => panic!("invalid chain spec"),
 						}
 					} else if cfg!(feature = "moonbase-runtime-benchmarks") {
+						#[cfg(feature = "moonbase-native")]
```

Comment:
Having `moonbase-runtime-benchmarks` enabled also enables `moonbase-native`.
https://github.com/moonbeam-foundation/moonbeam/blob/c155768dfa954147e65b678551b232a325c745bb/node/service/Cargo.toml#L229-L231

But is better to be explicit.

Reply:
Looking into the code, it seems that we can get rid of `moonbase-runtime-benchmarks`. Will create an issue for this, should be an easy task.

The PR changes look good, will approve it once green.

Reply:
Feel free to edit this PR. I've bootstrapped it as it is an easy one but if there is more to add, go ahead